cpp/README.md

```diff
@@ -324,6 +324,9 @@ _bob_ sets the compilation flags according to a set of CMake variables:
 * `${PROJECT_NAME}_NORMAL_CXX_FLAGS:BOOL`: Allow `CMAKE_CXX_FLAGS` to follow _normal_ CMake behavior
   and bypass all variables above.
 
+Alternatively you can set `${PROJECT_NAME}_CODING_CONV_PREFIX:BOOL` to a user defined string to
+prefix above bob compilation flags.
```

Comment:
Shouldn't the variable be declared before importing the submodule? Don't you also have to force the value in your case? something like:

```cmake
set(${PROJECT_NAME}_CODING_CONV_PREFIX "foo" "alternative prefix " CACHE STRING FORCE)
 add_subdirectory(hpc-coding-conventions/cpp)
```

Reply:
I was just mucking with this and it seems to be OK to pass this as a user defined argument since it's defined first. If you want to set it in your project then you have to do it before you include hpc-coding-conventions. That's how i did it in coreneuron

```cmake
set(coreneuron_CODING_CONV_PREFIX
        "CORENRN"
        CACHE STRING "Override coding conventions options prefix (default: PROJECT_NAME)")
add_subdirectory(CMake/hpc-coding-conventions/cpp)
```